src/Model/Consignment/BpostConsignment.php

```diff
@@ -247,8 +247,8 @@ public function setRetailNetworkId(string $retailNetworkId): AbstractConsignment
      */
     public function validate(): bool
     {
-        if ($this->getTotalWeight() < 50) {
-            throw new InvalidConsignmentException('It is necessary to at a minimum weight of 50 grams');
+        if ($this->getTotalWeight() < 1) {
+            throw new InvalidConsignmentException('It is necessary to at a minimum weight of 1 grams');
```

Comment:
We hadden we volgens mij gedaan zodat we zeker weten dat iemand niet kg meegeeft.